### PR TITLE
Add ability to hide apps count

### PR DESCRIPTION
**Pull request by FikriRNurhidayat**
#### What is it?
- [ ] Bugfix (user facing)
- [X] Feature request (user facing)
- [ ] Codebase improvement (dev facing)

#### Description of the changes in your PR

Added new option on Apps to hide Apps Count

### CODE REVIEW

File: app/src/main/kotlin/rasel/lunar/launcher/apps/AppsAdapter.kt [hunk 1]
```diff
@@ -138,6 +140,15 @@ internal class AppsAdapter(
             notifyDataSetChanged()
         }
     }
+
+    /* update apps count visible */
+    @SuppressLint("RtlHardcoded", "NotifyDataSetChanged")
+    fun updateAppsCountVisible(value: Boolean){
+        if (appsCountVisible != value) {
+            appsCountVisible = value
+            notifyDataSetChanged()
```
Comment (iamrasel) [inline app/src/main/kotlin/rasel/lunar/launcher/apps/AppsAdapter.kt:149]: No need to check for the change here.

File: app/src/main/kotlin/rasel/lunar/launcher/settings/childs/Apps.kt [hunk 2]
```diff
@@ -111,6 +119,19 @@ internal class Apps : BottomSheetDialogFragment() {
 
         return binding.root
     }
+    @SuppressLint("RtlHardcoded")
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
```
Comment (iamrasel) [inline app/src/main/kotlin/rasel/lunar/launcher/settings/childs/Apps.kt:123]: `onCreateDialog` is called before `onCreateView`

Comment (FikriRNurhidayat) [reply app/src/main/kotlin/rasel/lunar/launcher/settings/childs/Apps.kt]: Thanks, I'm not really familiar with Android or Kotlin in General, but I was just hacking through it to get desired result on my end.

Comment (iamrasel) [reply app/src/main/kotlin/rasel/lunar/launcher/settings/childs/Apps.kt]: I am also doing the same thing, along with the help of web. :smiley:

**Pull request merged**